paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -44,12 +44,14 @@ fun detectEnvironment(): Environment {
       .reduce { _, next -> next }
       .orElse(null)
 
-  val configLines = File("build/intermediates/paparazzi/resources.txt").readLines()
-  val packageName = configLines[0]
-  val resDir = configLines[1]
-  val compileSdkVersion = configLines[2].toInt()
+  File(System.getProperty("PAPARAZZI_RESOURCES_DETAILS_FILE_KEY")).readLines()
```

Comment:
by taking the path from the outside, it allows us to use other `build-dir` values within Gradle and also different build systems.